src/php/ext/grpc/channel.c

```diff
@@ -431,6 +434,7 @@ PHP_METHOD(Channel, close) {
   gpr_mu_lock(&channel->wrapper->mu);
   if (channel->wrapper->wrapped != NULL) {
     grpc_channel_destroy(channel->wrapper->wrapped);
+    free(channel->wrapper->target);
```

Comment:
The `key` is used to search through the list, not the `target`, which is part of the `key`